doc/admin-guide/plugins/access_control.en.rst

```diff
@@ -0,0 +1,464 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+
+.. include:: ../../common.defs
+
+.. _admin-plugins-access_control:
+
+
+Access Control Plugin
+*********************
+
+Description
+===========
+The `access_control` plugin covers common use-cases related to providing access control to the objects stored in CDN_ cache.
+
+
+Requirements and features
+=========================
+
+#. _`Cache Access Control` - cached objects to be served only to properly authenticated and authorized users.
+#. _`Authorizing Multiple Requests` - all requests from a particular UA_ in a defined time period to be authenticated and authorized only once.
+#. _`Multiple Page Versions` - objects may have the same name (URI) but different content for each `target audience`_ and must be served only to the corresponding `target audience`_.
+#. _`Proxy Only Mode` - CDN_ proxies the request to the origin_. In case of access control failure at the Edge_, the UA_ would not be redirected to other services, the failed request would be forwarded to the origin_.
+
+Participants
+============
+
+* _`UA` - User Agent used by a user whose requests are subject to access control.
+* _`Target audience` - an application specific audience (role, group of users, etc) to which the authenticated and authorized user belongs and which defines user's permissions.
+* _`CDN` / _`Edge` - Content Delivery Network, sometimes CDN and Edge are used interchangeably
+* _`IdMS` - Identity Management Services
+* _`DS` - Directory services
+* _`Origin` / _`Application` - origin server which is part of the particular application, sometimes origin and application are used interchangeably
+
+
+Design considerations and limitations
+=====================================
+
+The following are some thoughts and ideas behind the access control flow and the plugin design.
+
+
+* **Existing standards** - OAuth [1]_ [2]_ and SAML [3]_ were considered looking for use cases matching standard-based solutions but none seemed to match the requirements completely, also supporting some of the mentioned standards would require a more involved design and implementation.
```

Comment:
For formatting, you might try something like
```
**Existing standards** 
   OAuth [1]_ [2]_ and SAML [3]_ were considered looking for use cases matching standard-based solutions but none 
   seemed to match the requirements ...
```
This will give you a header with an indented following paragraph. [See here](https://docs.trafficserver.apache.org/en/latest/developer-guide/internal-libraries/buffer-writer.en.html#specialized-types) for an example.